Automattic/newspack-plugin
# Update the `Modal` component by removing Muriel and match `Card`

### All Submissions:






### Changes proposed in this Pull Request:

* Remove Muriel from `Modal`
* Match the style of `Card`
* Add a specific class to `Card` when it's used as a buttons container and update Components Demo accordingly

__Before:__

![1 before](https://user-images.githubusercontent.com/177929/71203707-6a2d1680-2296-11ea-8384-8d62923ecd2c.png)

__After:__

![2 after](https://user-images.githubusercontent.com/177929/71203698-66998f80-2296-11ea-9e22-917f3343d54c.png)

### How to test the changes in this Pull Request:

1. Go to Components Demo.
2. Check out the "Open modal".
3. Switch to this branch.
4. Does it work as expected? Does it look like a `Card`?

### Other information:
